Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor
+{
+    internal class BlenderLauncher : IDCCLauncher
+    {
+        const string menuItem_OpenInBlender = "Assets/MeshSync/Open in blender";
+
+        const string editorSettingPath = "MESHSYNC_BLENDER_PATH";
+
+        System.Diagnostics.Process m_blenderProcess;
+        static bool redirectBlenderToUnityConsole;
+
+        public RunMode runMode { get; set; }
+
+        ~BlenderLauncher()
+        {
+            Cleanup();
+        }
+
+        static string GetBlenderPath()
+        {
+            var blenderPath = EditorPrefs.GetString(editorSettingPath);
+
+            if (string.IsNullOrEmpty(blenderPath))
+            {
+                if (!DefaultAppUtility.TryGetRegisteredApplication(".blend", out blenderPath))
+                {
+                    if (Application.platform == RuntimePlatform.OSXEditor)
+                    {
+                        blenderPath = "/Applications/Blender.app/Contents/MacOS/Blender";
+                    }
+                }
+
+                SetBlenderPath(blenderPath);
+            }
+
+            return blenderPath;
+        }
+
+        public static void SetBlenderPath(string blenderPath)
+        {
+            EditorPrefs.SetString(editorSettingPath, blenderPath);
+        }
+
+        [MenuItem(menuItem_OpenInBlender, priority = 0)]
```

Comment:
Please put the priority lower, as top priority menuitems are reserved for standard Unity menus.

Please also put menuitem functions in `MeshSyncMenu.cs`

Reply:
Done